_config.yml

```diff
@@ -192,6 +192,10 @@ component_releases:
 
 release_archive:
     flink:
+      -
+        version_short: "1.10"
```

Comment:
This is right but it would be better to make it consistent with others, i.e., remove the quotes.

Reply:
Thanks for the review. The quotes are for preventing issues like reported in FLINK-16663, FYI.